commands/wrappers.go

```diff
@@ -27,24 +27,30 @@ import (
 
 var (
 	composerWrapper = &console.Command{
-		Name:   "composer",
 		Usage:  "Runs Composer without memory limit",
 		Hidden: console.Hide,
+		// we use an alias to avoid the command being shown in the help but
+		// still be available for completion
+		Aliases: []*console.Alias{{Name: "composer"}},
 		Action: func(c *console.Context) error {
 			return console.IncorrectUsageError{ParentError: errors.New(`This command can only be run as "symfony composer"`)}
 		},
 	}
 	binConsoleWrapper = &console.Command{
-		Name:   "console",
 		Usage:  "Runs the Symfony Console (bin/console) for current project",
 		Hidden: console.Hide,
+		// we use an alias to avoid the command being shown in the help but
```

Comment:
It's hidden anyway, no?

Reply:
"yes... but"

with https://github.com/symfony-cli/console/pull/17, the alias will be considered for autocompletion. which means `symfony co` + tab will now suggest `symfony console`